src/scripts/main.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const mes = document.createElement('div');
+
+  mes.style.top = `${posTop}px`;
+  mes.style.right = `${posRight}px`;
+
+  mes.classList.add('notification');
+  mes.classList.add(type);
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.textContent = title;
+
+  mes.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+
+  mes.appendChild(descriptionElement);
+
+  document.body.appendChild(mes);
+
+  setTimeout(() => {
+    mes.style.display = 'none';
```

Comment:
Hiding the notification by setting `display` to 'none' does not remove it from the DOM. Consider using `mes.remove()` to completely remove the element after it is hidden to prevent memory leaks.